### ISSUE TITLE
Standardized names -- Joe Kennedy III (MA)

### ISSUE THREAD

**Issue by bellodi92** [kuriwaki/cces_candidates#35]
Great data, guys! Thank you for all the work you've put to assemble this dataset.

I think there's a candidate with two different name_snyder entries whereas in fact they're the same candidate.
KENNEDY, JOSEPH P. (JOE), III
KENNEDY, JOSEPH P., III

Also, there are three "rows" for DEBICELLA, DAN running in 2014, CT, 4th District.

**Comment by kuriwaki** [comment]
Thanks. The Kennedy one is an error we should fix. The Debicella one was fixed internally in #28 but it's not up in the Dropbox. We'll upload these accumulated fixes as Dataverse V2 soon.

**Comment by bellodi92** [comment]
I think I've spotted a couple of other mistakes. These seem to be the same candidates:
"BEUTLER, JAIME HERRERA" -- "HERRERA BEUTLER, JAIME"
"CUNNANE, MADELEINE DEAN" -- "DEAN, MADELEINE"
"WAKELY, TOMMY" -- "WAKELY, THOMAS J. (TOM)"

**Comment by kuriwaki** [comment]
Fixed internally in #37 -- it currently looks like this, checking that there is only one version of each name Will merge when we upload v2 to dataverse

``` r
library(haven)
library(dplyr)
library(stringr)

dat <- read_dta("release/candidates_2006-2020.dta")
dat |> 
  filter(str_detect(name_snyder, "(KENNEDY, JOSEPH|WAKELY, T|BEUTLER|CUNNANE)")) |> 
  count(name_snyder)
#> # A tibble: 4 × 2
#>   name_snyder                       n
#>   <chr>                         <int>
#> 1 DEAN CUNNANE, MADELEINE           2
#> 2 HERRERA BEUTLER, JAIME            6
#> 3 KENNEDY, JOSEPH P. (JOE), III     4
#> 4 WAKELY, THOMAS J. (TOM)           2
```

<sup>Created on 2023-06-14 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>
